src/features/content-explorer/content-explorer/ContentExplorerFolderTreeBreadcrumbs.js

```diff
@@ -20,42 +20,49 @@ import {
 
 const ContentExplorerFolderTreeBreadcrumbs = ({
     foldersPath,
+    hideFolderTreeButton,
     intl: { formatMessage, formatNumber },
     isFolderTreeButtonDisabled = false,
     numTotalItems,
     onBreadcrumbClick,
 }) => {
     const currentFolderName = foldersPath[foldersPath.length - 1].name;
+    const foldersPathWithoutLast = foldersPath.slice(0, -1);
     return (
         <div className="bdl-ContentExplorerFolderTreeBreadcrumbs">
-            <DropdownMenu>
-                <Button
-                    aria-label={formatMessage(messages.clickToViewPath)}
-                    className="bdl-ContentExplorerFolderTreeBreadcrumbs-button"
-                    isDisabled={isFolderTreeButtonDisabled}
-                    title={formatMessage(messages.filePath)}
-                    type="button"
-                >
-                    <IconFolderTree height={FOLDER_TREE_ICON_HEIGHT} width={FOLDER_TREE_ICON_WIDTH} />
-                </Button>
-                <Menu>
-                    {foldersPath.map((folder, i) => (
-                        <MenuItem
-                            data-testid="folder-tree-item"
-                            key={folder.id}
-                            onClick={event => onBreadcrumbClick(i, event)}
+            {!hideFolderTreeButton && (
+                <div className="bdl-ContentExplorerFolderTreeBreadcrumbs-dropdown-icon-container">
```

Comment:
descendants in SUIT naming should use camelcase: https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#componentname-descendentname

not sure if this is a better solution but you could individually conditionally hide both components. this way you won't need the wrapping div and style updates:
```
<div className="bdl-ContentExplorerFolderTreeBreadcrumbs">
        {!hideFolderTreeButton && <DropdownMenu [...] />}
        {!hideFolderTreeButton && <Arrow16 [...] />}
        <span className="bdl-ContentExplorerFolderTreeBreadcrumbs-text" [...] />
```